src/core/view/LayerView.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Xournal++
+ *
+ * Displays a Layer
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#pragma once
+
+#include "util/Rectangle.h"
+
+#include "View.h"
+
+class Layer;
+
+class xoj::View::LayerView {
+public:
+    LayerView(const Layer* layer);
+    ~LayerView();
```

Comment:
Don't need to declare (and define) the destructor if the default one is being used.